lib/rules/accessor-pairs.js

```diff
@@ -66,80 +66,84 @@ function isPropertyDescriptor(node) {
 // Rule Definition
 //------------------------------------------------------------------------------
 
-module.exports = function(context) {
-    var config = context.options[0] || {};
-    var checkGetWithoutSet = config.getWithoutSet === true;
-    var checkSetWithoutGet = config.setWithoutGet !== false;
-
-    /**
-     * Checks a object expression to see if it has setter and getter both present or none.
-     * @param {ASTNode} node The node to check.
-     * @returns {void}
-     * @private
-     */
-    function checkLonelySetGet(node) {
-        var isSetPresent = false;
-        var isGetPresent = false;
-        var isDescriptor = isPropertyDescriptor(node);
-
-        for (var i = 0, end = node.properties.length; i < end; i++) {
-            var property = node.properties[i];
-
-            var propToCheck = "";
-            if (property.kind === "init") {
-                if (isDescriptor && !property.computed) {
-                    propToCheck = property.key.name;
-                }
-            } else {
-                propToCheck = property.kind;
+module.exports = {
+    meta: {
+        errors: {
+            getterMissing: "Getter is not present",
+            setterMissing: "Setter is not present"
+        },
+        description: "enforces getter/setter pairs in objects",
+        ruleType: "Best Practices",
+        recommended: false,
+        fixable: false,
```

Comment:
On the other hand, being able to know ahead of time if a rule is going to try to apply fixes could be quite useful. For instance, we might want to apply rules that have fixes in one pass, then do all the others to make sure we didn't miss anything.

We could make it so you can only specify `fix` as part of `context.report()` if the rule has declared itself `fixable` (otherwise, throw an error). We'd obviously only be able to do this for new-style rules, but could be nice.